db/fixtures/003_flags.rb

```diff
@@ -1,9 +1,16 @@
 # frozen_string_literal: true
 
+Flag.seed do |s|
+  s.id = 6
+  s.name = "notify_user"
+  s.notify_type = false
+  s.auto_action_type = false
+  s.custom_type = true
+  s.applies_to = %w[Post Chat::Message]
+end
 Flag.seed do |s|
   s.id = 3
   s.name = "off_topic"
-  s.position = 2
```

Comment:
I guess it doesn't matter because we have the migration too, but why not keep the position here as well?

Reply:
This was actually a bug :( . Admin can change position in UI and we don't want seed task with next deploy to override it again back to original position.

Reply:
Oh I thought the seeds only ran once if the ID already existed :thinking: Guess not! Or does it do a INSERT ON CONFLICT UPDATE maybe?